components/Op/OpAboutPanel.js

```diff
@@ -5,24 +5,43 @@ import TagDisplay from '../Tags/TagDisplay'
 import Html from '../VTheme/Html'
 import { ItemIdLine, TagContainer } from '../VTheme/ItemList'
 import { ProfilePanel, ProfileSection } from '../VTheme/Profile'
-
+import { OpSectionGrid, Spacer } from '../VTheme/VTheme'
+import { Divider } from 'antd'
+import { ShareLinks } from './OpShareLinks'
+import { config } from '../../config/config'
+import { FormattedMessage } from 'react-intl'
 export function OpAboutPanel ({ op }) {
   const description = op.description || ''
-
+  const appUrl = `${config.appUrl}/ops/op._id`
   return (
     <ProfilePanel>
-      <ProfileSection>
-        <ItemIdLine item={op.requestor} path='people' />
-        <TagContainer>
-          <TagDisplay tags={op.tags} />
-        </TagContainer>
-      </ProfileSection>
-      <ProfileSection>
-        <Html>
-          {description}
-        </Html>
-      </ProfileSection>
 
+      <OpSectionGrid>
+        <h2><FormattedMessage id='actDetailForm.AboutSection.subtitle' /></h2>
+        <div>
+          <Html>
+            {description}
+          </Html>
+          <Divider />
+          <TagContainer>
+            <h5><FormattedMessage id='actCategories' /></h5>
+            <TagDisplay tags={op.tags} />
+          </TagContainer>
+
+          <Divider />
+          <h5><FormattedMessage id='actShare' /></h5>
```

Comment:
FormattedMessage MUST have a defaultMessage.

This is a problem as it uses an id without providing a defaultMessage or description. Only Adding it to en.json will cause a break when the language file is regenerated.

Reply:
Fixed in latest commit